plugins/mbaas/README.md

```diff
@@ -0,0 +1,4 @@
+
+Contents in here will overlap on top of the files from the default subtree
+
+*Note: Any files with the same sub path and name as a file from default will be overwritten with the one here (This allows for core/mbaas to override the behavior we have in default)*
```

Comment:
We have a future task to allow using the image for mbaas or core. Isn't this README a sign that maybe we should defer to a different method, and not be merging directory trees?